pkg/ddevapp/config.go

```diff
@@ -31,6 +33,9 @@ const DDevTLD = "ddev.local"
 // AllowedAppTypes lists the types of site/app that can be used.
 var AllowedAppTypes = []string{"drupal7", "drupal8", "wordpress"}
 
+// Regexp pattern for alphanumeric + dashes
+var re = regexp.MustCompile(`[^a-zA-Z0-9_-]+`)
```

Comment:
I think I misunderstood the intent - I was giving a validation string, not a replacement string.